packages/admin-ui/src/filters/filter-base.tsx

```diff
@@ -0,0 +1,72 @@
+import type { ReactNode } from 'react'
+import React, { useRef } from 'react'
+
+import { Button } from '../button'
+import { FilterPopoverFooter, FilterPopover } from './filter-popover'
+import { FilterDisclosure } from './filter-disclosure'
+
+import type { GenericFilterStateReturn } from './filter.state'
+import { ComboboxList } from 'ariakit'
+
+import { tag } from '@vtex/admin-ui-react'
+
+import { useMessageFormatter } from '../i18n'
+import { messages } from './filter.i18n'
+
+export function BaseFilter(props: BaseFilterProps) {
+  const { state, children, appliedValuesLabel } = props
+  const { onClear, onChange, label, menu, combobox } = state
+
+  const formatMessage = useMessageFormatter(messages.actions)
+
+  const optionsContainerRef = useRef<HTMLDivElement>(null)
+
+  return (
+    <>
+      <FilterDisclosure state={menu}>
+        {label}
+        {appliedValuesLabel || ''}
+      </FilterDisclosure>
+
+      <FilterPopover state={menu}>
+        <tag.span ref={optionsContainerRef}>
```

Comment:
i think a `div` fits better than span in this case, because it is a container of a items list 🤔